ghostinspector.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Deployer;
+
+desc('Run Ghostinspector test suite');
+task('ghostinspector:run', function () {
+    $config = get('ghostinspector', []);
+    if (!isset($config['version'])) {
```

Comment:
This could be dropped if defaults are used as suggested above. People can then simply `add()` to set `apikey` and `testsuite`.